ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -470,6 +446,34 @@
                                                              earlier-date
                                                              later-date)))))))
 
+(define-event LoadServiceChangesForDateResponse [response detection-date]
+  {}
+  (let [date-filter (if (= "now" (:scope app))
+                      (time/now-iso-date-str)
+                      detection-date)
+        route-hash (js/decodeURIComponent (get-in app [:params :route]))
+        changes (future-changes date-filter (:route-changes response))
+        route (some
+                #(when
+                  (= (str/replace (:route-hash-id %) #"\s" "") route-hash)
+                  %)
+                changes)]
+    (when route-hash                                        ;;route-hash exists when you have a url where route is selected
```

Comment:
Should the condition check for `route` instead of `route-hash`? 
Because `route-hash` is not referred within the block, but `route ` is.